crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -87,6 +111,82 @@ class CrowdfundingProject(models.Model):
     owner_firstname = fields.Char(string="Your firstname")
     active = fields.Boolean(default=True)
 
+    @api.one
+    @api.depends('cover_photo')
+    def _compute_images(self):
+        if self._context.get('bin_size'):
+            self.image_medium = self.cover_photo
+            self.image_small = self.cover_photo
+            self.image = self.cover_photo
+        else:
+            resized_images = tools.image_get_resized_images(self.cover_photo, return_big=True,
+                                                            avoid_resize_medium=True)
+            self.image_medium = resized_images['image_medium']
+            self.image_small = resized_images['image_small']
+            self.image = resized_images['image']
+
+    @api.one
```

Comment:
api.one is deprecated, it would be good to remove it and make the method like the other computes with a loop :
```
for project in self:
    project.cover_photo = project.image_medium
....
```